tests/unit/web/ng/ResponseTests.cpp

```diff
@@ -150,7 +156,9 @@ TEST_F(ResponseTest, createFromStringAndConnection)
 
 TEST_F(ResponseTest, createFromJsonAndConnection)
 {
-    util::TagDecoratorFactory const tagDecoratorFactory{util::Config{}};
+    util::TagDecoratorFactory tagDecoratorFactory{
```

Comment:
Can't be `const` anymore?

Reply:
should definitely be const 💯 . Im going to let clang-tidy fix it, instead of pushing and needing approval again